axelrod/strategies/axelrod_first.py

```diff
@@ -56,97 +69,202 @@ def strategy(self, opponent: Player) -> Action:
+    always playing D against the estimage probabilities. This corresponds to:
+
+        a. In the case of the player always cooperating:
 
-class RevisedDowning(Player):
-    """This strategy attempts to estimate the next move of the opponent by estimating
-    the probability of cooperating given that they defected (:math:`p(C|D)`) or
-    cooperated on the previous round (:math:`p(C|C)`). These probabilities are
-    continuously updated during play and the strategy attempts to maximise the long
-    term play. Note that the initial values are :math:`p(C|C)=p(C|D)=.5`.
+           P_CC = alpha and P_CD = 1 - alpha
 
-    Downing is implemented as `RevisedDowning`. Apparently in the first tournament
-    the strategy was implemented incorrectly and defected on the first two rounds.
-    This can be controlled by setting `revised=True` to prevent the initial defections.
+        b. In the case of the player always defecting:
 
-    This strategy came 10th in Axelrod's original tournament but would have won
-    if it had been implemented correctly.
+           P_DC = beta and P_DD = 1 - beta
+
+
+    Using this we have:
+
+        E_C = alpha R + (1 - alpha) S
+        E_D = beta T + (1 - beta) P
+
+    Thus at every turn, the strategy will calculate those two values and
+    cooperate if E_C > E_D and will defect if E_C < E_D.
+
+    In the case of E_C = E_D, the player will alternate from their previous
+    move. This is based on specific sentence from Axelrod's original paper:
+
+    > "Under certain circumstances, DOWNING will even determine that the best
+    > strategy is to alternate cooperation and defection."
+
+    One final important point is the early game behaviour of the strategy. It
+    has been noted that this strategy was implemented in a way that assumed that
+    alpha and beta were both 1/2:
+
+    > "Initially, they are both assumed to be .5, which amounts to the
+    > pessimistic assumption that the other player is not responsive."
+
+    Thus, the player opens with a defection in the first two rounds. Note that
+    from the Axelrod publications alone there is nothing to indicate defections
+    on the first two rounds, although a defection in the opening round is clear.
+    However there is a presentation available at
+    http://www.sci.brooklyn.cuny.edu/~sklar/teaching/f05/alife/notes/azhar-ipd-Oct19th.pdf
+    That clearly states that Downing defected in the first two rounds, thus this
+    is assumed to be the behaviour.
+
+    Note that response to the first round allows us to estimate
+    beta = P(C_o | D_s) and we will use the opening play of the player to
+    estimate alpha = P(C_o | C_s). This is an assumption with no clear
+    indication from the literature.
+
+    --
+    This strategy came 10th in Axelrod's original tournament.
 
     Names:
 
     - Revised Downing: [Axelrod1980]_
     """
 
-    name = "Revised Downing"
+    name = "First tournament by Downing"
```

Comment:
How do you feel about "First by Downing"? Just considering how the longer names will appear in plots.

Reply:
Yup completely fine by me.